### PR TITLE
Expose linter errors in red

### PR DESCRIPTION
**Pull request by howardjohn**
Currently they are really hard to see

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
I'm not so worried about the individual messages (I guess I've trained myself to know where to look). What I would really like is for the spelling error highlighting to be visible in the logs similar to running locally. Here's what it might look like locally (not that the spelling error here aren't valid), just wanting to show the highlighting.
Screen Shot 2022-09-07 at 11 08 29 AM

**Comment by howardjohn** [comment]
My concern was running locally. I spent a long time staring at the console
not understanding what was going on before I found the error in the huge
log of output

**Comment by ericvn** [comment]
OK. Would then want to make sure that the making of the output red doesn't negatively impact the spelling highlighting.

File: scripts/lint_site.sh [hunk 1]
```diff
@@ -24,6 +24,13 @@ else
     LANGS="en zh"
 fi
 
+red='\e[0;31m'
+clr='\e[0m'
+
+error() {
+  echo -e "${red}$@${clr}"
```
Comment (ericvn) [inline scripts/lint_site.sh:31]:
```suggestion
  echo -e "${red}$*${clr}"
```
to fix the lint error, and allow `make lint` to run.

**Comment by ericvn** [comment]
> OK. Would then want to make sure that the making of the output red doesn't negatively impact the spelling highlighting.

Screen Shot 2022-09-07 at 1 47 58 PM

Seems to be OK.

**Review by ericvn — APPROVED** [review]

**Pull request merged**